spec/requests/invites_controller_spec.rb

```diff
@@ -102,6 +102,14 @@
 
       before { sign_in(user) }
 
+      it "automatically redirects to the topic if the user can access it" do
```

Comment:
Can we add a test case for "does not redirect if user does not have access to topic"?

Reply:
Yup, was waiting for confirmation this was the behaviour we wanted before writing moar specs. I've pushed a spec for the "can't access topic" case 👍